pkgs/by-name/ag/age-plugin-se/package.nix

```
@@ -0,0 +1,63 @@
+{
+  lib,
+  fetchFromGitHub,
+  swiftPackages,
+  swift,
+  swiftpm,
+  nix-update-script,
+}:
+let
+  stdenv = swiftPackages.stdenv;
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "age-plugin-se";
+  version = "0.1.4";
+
+  src = fetchFromGitHub {
+    owner = "remko";
+    repo = "age-plugin-se";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-sg73DzlW4aXNbIIePZox4JkF10OfsMtPw0q/0DWwgDk=";
+  };
+
+  nativeBuildInputs = [
+    swift
+    swiftpm
+  ];
+
+  patchPhase =
```

```suggestion
  postPatch  =
```
otherwise we break patches